src/GrpcProtos/capture.proto

```diff
@@ -325,6 +325,24 @@ message SystemMemoryUsage {
   int64 cached_kb = 6;
 }
 
+message ProcessMemoryUsage {
+  uint32 pid = 1;
+  uint64 timestamp_ns = 2;
+  // These fields are retrieved from /proc/<pid>/status.
+  int64 rss_kb = 3;
+}
+
+message CGroupMemoryUsage {
+  string cgroup_name = 1;
+  uint64 timestamp_ns = 2;
+  // This field is retrieved from
+  // /sys/fs/cgroup/memory/<cgroup_name>/memory.stat.
+  int64 rss_kb = 3;
```

Comment:
Since the original value in the memory.stat file is in bytes, I would also keep the value in bytes. The rounding practically will not make a difference for users, but it's easier to avoid mistakes when parsing the data and when someone else needs to touch the code in the future. Reading this here, it's easy to assume that the value in the memory.stat file is in kb.

Reply:
Totally agree that use the same unit size as the original value would be less misleading! Changed to use `rss_bytes` here. Thanks!